README.md

```diff
@@ -1,10 +1,11 @@
 
-# scikit-learn_bench
+# Machine Learning Benchmarks
 
-**scikit-learn_bench** benchmarks various implementations of machine learning algorithms across data analytics frameworks.  Scikit-learn_bench can be extended to add new frameworks and algorithms.  It currently support the [scikit-learn](https://scikit-learn.org/), [DAAL4PY](https://intelpython.github.io/daal4py/), [cuML](https://github.com/rapidsai/cuml), and [XGBoost](https://github.com/dmlc/xgboost) frameworks for commonly used [machine learning algorithms](#supported-algorithms).
+**Machine Learning Benchmarks** benchmarks various implementations of machine learning algorithms across data analytics frameworks.  Scikit-learn_bench can be extended to add new frameworks and algorithms.  It currently support the [scikit-learn](https://scikit-learn.org/), [DAAL4PY](https://intelpython.github.io/daal4py/), [cuML](https://github.com/rapidsai/cuml), and [XGBoost](https://github.com/dmlc/xgboost) frameworks for commonly used [machine learning algorithms](#supported-algorithms).
 
-See benchmark results [here](https://intelpython.github.io/scikit-learn_bench).
+## Our articles
 
+https://medium.com/intel-analytics-software
```

Comment:
Better to use this link:
https://medium.com/intel-analytics-software/tagged/machine-learning

Because yours shows only latest.

Reply:
Thank you!